updateprotos.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This is to be called manually?

Reply:
Yup. (Like generateprotos.sh is today. We do this on an ad-hoc basis.)